### ISSUE TITLE
[Bug] Different dashboard options on different PC's

### ISSUE THREAD

**Issue by ManCaveMedia** [WDaan/VueTorrent#237]
On one of my PC's I have far more options for settings the dashboard order than on another.
(Downloaded, Uploaded, Availability and some more)
I think it's due to it storing previous settings and not updating the settings with new options.

Is there a way to remove or reset the settings so newer options on the dashboard become available?

**Comment by m4ximuel** [comment]
VueTorrent does not act as a server. Therefore, VueTorrent's own settings cannot be shared between other PCs.

**Comment by ManCaveMedia** [comment]
OK, I understand that.
But where are the settings stored. I've deleted several cookies on my laptop but I'm still not able to see/sort/ enable the mentioned options (Downloaded, Uploaded, Availability and some more)

When I open the URL on a different browser on the same laptop or in the same browser on a different PC the options are there...

**Comment by ManCaveMedia** [comment]
OK, nevermind...
It took me a couple of times to find the cookie. Since I'm using a reverse proxy and DNS settings I had to find al (previously used) versions of the cookie to delete them all. Seems to be working correctly now.

But is still weird that I never saw the newer options/labels on the dashboard settings when they were added.
I'll leave it up to you to determine whether this needs a fix.
